rustls/tests/api.rs

```diff
@@ -798,21 +787,23 @@ fn test_tls13_late_plaintext_alert() {
 
     // Inject a plaintext alert from the client. The server should attempt to decrypt this message.
     server
-        .read_tls(&mut io::Cursor::new(
-            PlainMessage::from(Message::build_alert(
-                AlertLevel::Fatal,
-                AlertDescription::UnknownCA,
-            ))
-            .borrow_inbound()
-            .to_unencrypted_opaque()
-            .encode(),
-        ))
+        .read_tls(&mut io::Cursor::new(&build_alert(AlertLevel::Fatal, AlertDescription::UnknownCA, &[])))
         .unwrap();
 
     // The server should produce a decrypt error, trying to decrypt a plaintext alert.
     assert_eq!(server.process_new_packets(), Err(Error::DecryptError));
 }
 
+fn build_alert(level: AlertLevel, desc: AlertDescription, suffix: &[u8]) -> Vec<u8> {
+    let mut v = vec![ ContentType::Alert.get_u8() ];
```

Comment:
Not sure why were not using `Message::build_alert()` here anymore?

Reply:
`Message::build_alert` can only make syntactically valid messages, but this function is less restrictive. An additional side-effect of this change is to remove the only use of `to_unencrypted_opaque()` on `InboundMessage`.

Reply:
Fair!